content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -234,10 +234,10 @@ These pods are tracked via `.status.currentHealthy` field in the PDB status.
 
 ## Unhealthy Pod Eviction Policy
 
-{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 {{< note >}}
-In order to use this behavior, you must enable the `PDBUnhealthyPodEvictionPolicy`
+This feature is enabled by default. You can disable it by disabling the `PDBUnhealthyPodEvictionPolicy`
```

Comment:
```suggestion
This feature is enabled by default. You can disable it by setting the `PDBUnhealthyPodEvictionPolicy=false`
```

Reply:
To me, “disabling the `PDBUnhealthyPodEvictionPolicy` feature gate” is idiomatic. I don't recommend making this change.

For example, if we one day allow managing API server feature gates with a configuration file, disabling a gate is actually more likely to look like:
```yaml
   …
   - PDBUnhealthyPodEvictionPolicy: false # colon, not equality symbol
   …
```

Reply:
That's even better suggestion :+1: